### PR TITLE
Stop People from spamming with the !stats command

### PR DESCRIPTION
**Pull request by Arzte**
"They didn't listen to me, so I made them listen"

### CODE REVIEW

**Comment by Arzte** [comment]
Don't merge just quite yet, gonna squash the extra merge commits.

File: Commands/Commands/utils.js [hunk 1]
```diff
@@ -79,7 +78,7 @@ commands.stats = {
           inline: true
         }
       )
-      msg.channel.sendMessage('', false, {
+      msg.channel.openDM('', false, {
```
Comment (Dougley) [inline Commands/Commands/utils.js]: `openDM()` returns a channel as a promise resolve, the way you're trying to do it now won't work.

**Review by Dougley — DISMISSED** [review]

File: Commands/Commands/utils.js (hunk 1 above)
Comment (Arzte) [reply Commands/Commands/utils.js]: Wow, fail on my part, completely forgot about that, been so long. (I also wasn't expecting a review so suddenly)

File: Commands/Commands/utils.js [hunk 2]
```diff
@@ -79,14 +78,16 @@ commands.stats = {
...
-      msg.channel.sendMessage('', false, {
-        color: 0x59f442,
-        title: `${msg.author.username} - Statistics`,
-        thumbnail: {
-          url: msg.author.staticAvatarURL
-        },
-        fields: field
-      }).catch(bugsnag.notify)
+      msg.channel.openDM().then((e) => {
+        e.sendMessage('', false, {
```
Comment (Arzte) [inline Commands/Commands/utils.js:82]: huh, didn't pay attention to what I used for the data, I'd fix it if it actually mattered right now. (rewrite and all)

**Review by Dougley — APPROVED** [review]

**Pull request merged**